scripts/svm/addressToPublicKey.ts

```diff
@@ -0,0 +1,18 @@
+import { evmAddressToPublicKey } from "../../src/SvmUtils";
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+const argv = yargs(hideBin(process.argv)).option("address", {
+  type: "string",
+  demandOption: true,
+  describe: "Ethereum address to convert",
+}).argv;
+
+async function logPublicKey(): Promise<void> {
```

Comment:
Do we want to add this to Anchor scripts?

Reply:
can run it with node as well but I'll add it, sure.